central/scannerdefinitions/handler/handler.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/stackrox/rox/pkg/migrations"
 	"github.com/stackrox/rox/pkg/sync"
 	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/scanner/pkg/ziputil"
```

Comment:
I'm not sure what our stance is on using the scanner repo for more than just the clairify client library and gRPC stuff. Any thoughts @connorgorman ?